lib/Conversion/TritonGPUToLLVM/ConvertLayoutOpToLLVM.cpp

```diff
@@ -685,9 +685,6 @@ struct ConvertLayoutOpConversion
     auto srcTy = src.getType().cast<RankedTensorType>();
     auto srcShape = srcTy.getShape();
     auto dstTy = dst.getType().cast<RankedTensorType>();
-    auto dstShape = dstTy.getShape();
-    assert(dstShape.size() == 2 &&
```

Comment:
Should we modify the assert to <=2 instead of removing it ?

Reply:
Makes sense!